cfg/rke2-cis-1.8/master.yaml

```diff
@@ -204,10 +208,10 @@ groups:
 
       - id: 1.1.12
         text: "Ensure that the etcd data directory ownership is set to etcd:etcd (Automated)"
-        audit: "stat -c %U:%G /node/var/lib/etcd"
+        audit: "stat -c %U:%G /var/lib/rancher/rke2/server/db/etcd"
         tests:
           test_items:
-            - flag: "etcd:etcd"
+            - flag: "root:root"
```

Comment:
There is now a minor inconsistency with the `text` field here.
I think we should normalize it to a single expected owner and synchronize text/tests/remediation